nebula2/gclient/net/Connection.py

```diff
@@ -128,6 +151,29 @@ def execute_json(self, session_id, stmt):
                     raise IOErrorException(IOErrorException.E_UNKNOWN, te.message);
             raise
 
+    def execute_json_with_parameter(self, session_id, stmt, params):
+        """execute_json interface with session_id and ngql
+
+        :param session_id: the session id get from result of authenticate interface
+        :param stmt: the ngql
+        :return: string json representing the execution result
+        """
+        try:
+            resp = self._connection.executeJsonWithParameter(session_id, stmt)
+            return resp
+        except Exception as te:
+            if isinstance(te, TTransportException):
+                if te.message.find("timed out") > 0:
+                    self._reopen()
+                    raise IOErrorException(IOErrorException.E_TIMEOUT, te.message)
+                elif te.type == TTransportException.END_OF_FILE:
+                    raise IOErrorException(IOErrorException.E_CONNECT_BROKEN, te.message)
+                elif te.type == TTransportException.NOT_OPEN:
+                    raise IOErrorException(IOErrorException.E_NOT_OPEN, te.message)
+                else:
+                    raise IOErrorException(IOErrorException.E_UNKNOWN, te.message);
+            raise
```

Comment:
let non-parameter interface reuse the parameter interface.

Reply:
Fixed